### PR TITLE
Add custom environment variables from Java config xml file

### PR DESCRIPTION
**Pull request by cdonati**

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/456/

File: AppManager/app_manager_server.py [hunk 1]
```diff
@@ -6,15 +6,14 @@
 import logging
 import os
 import SOAPpy
-import socket
 import subprocess
 import sys
 import time
 import urllib
+from xml.etree import ElementTree
```
Comment (menivaitsi) [inline AppManager/app_manager_server.py]: Move this import after M2Crypto.

Comment (cdonati) [inline AppManager/app_manager_server.py]: I thought M2Crypto was separate because it was a custom pip install.

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/457/

File: AppManager/app_manager_server.py [hunk 2]
```diff
@@ -59,6 +58,15 @@
 # Load balancing path for datastore.
 DATASTORE_PATH = "localhost"
 
+class BadConfigurationException(Exception):
+  """ An application is configured incorrectly. """
+  def __init__(self, value):
```
Comment (nlake44) [inline AppManager/app_manager_server.py]: Have a doc string.

**Pull request merged**